web/modules/custom/dpl_react_apps/dpl_react_apps.module

```diff
@@ -43,6 +44,8 @@ function dpl_react_apps_preprocess_page(array &$variables): void {
         'string-suggestion-author-text' => t('Author', [], ['context' => 'Search Header']),
         'string-suggestion-topic-text' => t('Topic', [], ['context' => 'Search Header']),
         'string-suggestion-work-text' => t('Work', [], ['context' => 'Search Header']),
+        'header-dropdown-item-advanced-search-text' => t('Advanced search', [], ['context' => 'Search Header']),
+        'search-header-dropdown-text' => t('Dropdown with additional search functions', [], ['context' => 'Search Header']),
       // Add external API base urls.
```

Comment:
I reccomend ordering the text props alphabetically. It helps the further maintenance of it.